widgets-intro.md

```diff
@@ -344,8 +344,8 @@ widgets have
 [`onPressed`](https://docs.flutter.io/flutter/material/RaisedButton-class.html#onPressed)
 callbacks that are triggered when the user taps the widget.
 
-Managing state
---------------
+Managing state and handling input
```

Comment:
Yeah, we should remove "managing state" from the title.  That's the solution.  Instead, we should state the problem:

"Changing widgets in response to input" ?